[GitHub Actions] Use PHP 8.1 for phpDocumentor

takano32 opened the pull request:
**Description**

Use PHP 8.1 for phpDocumentor

fix this error: https://github.com/codeigniter4/CodeIgniter4/actions/runs/5889034105/job/15971401987


```
php tools/phpDocumentor run --ansi --verbose
  cp -R ${GITHUB_WORKSPACE}/source/api/build/* ${GITHUB_WORKSPACE}/api/docs
  shell: /usr/bin/bash -e {0}
  env:
    COMPOSER_PROCESS_TIMEOUT: 0
    COMPOSER_NO_INTERACTION: 1
    COMPOSER_NO_AUDIT: 1
Composer detected issues in your platform:

Your Composer dependencies require a PHP version ">= 8.1.0". You are running 8.0.29.

PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". You are running 8.0.29. in phar:///home/runner/.phive/phars/phpdocumentor-[3](https://github.com/codeigniter4/CodeIgniter4/actions/runs/5889034105/job/15971401987#step:8:3).[4](https://github.com/codeigniter4/CodeIgniter4/actions/runs/5889034105/job/15971401987#step:8:4).0.phar/vendor/composer/platform_check.php on line 24
Error: Process completed with exit code 2[5](https://github.com/codeigniter4/CodeIgniter4/actions/runs/5889034105/job/15971401987#step:8:5)5.
```
**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

takano32 commented:
@kenjis  Thanks! 🎉 


https://github.com/codeigniter4/CodeIgniter4/actions/runs/5889209744/job/15971920835

kenjis commented:
@takano32 Thank you! 
But there are many WARNINGs.
> 09:32:39 WARNING   [cache] Failed to save key "3c9acebe106b24f7af60f94bba86030e-569aef81d2de858070d4ee4198c6aa7b" of type string: fopen(phar:///home/runner/.phive/phars/phpdocumentor-3.4.0.phar/var/cache/prod/pools/app/5Jn5gw0nhP/2YnXc9-d): Failed to open stream: phar error: "var/cache/prod/pools/app/5Jn5gw0nhP/2YnXc9-d" is not a file in phar "/home/runner/.phive/phars/phpdocumentor-3.4.0.phar" ["key" => "3c9acebe106b24f7af60f94bba86030e-569aef81d2de858070d4ee4198c6aa7b","exception" => ErrorException { …},"cache-adapter" => "Symfony\Component\Cache\Adapter\FilesystemAdapter"]

These are not before. See https://github.com/codeigniter4/CodeIgniter4/actions/runs/5841093859/job/15840960974

takano32 commented:
I'll check how to disable cache in phpDocumentor.

takano32 commented:
@kenjis  I've read some documents about phpDocumentor, but I can't find the way to disable cahce.

* Config
   * https://docs.phpdoc.org/guide/references/full-config-reference.html
 * Cache
   * https://docs.phpdoc.org/guide/features/caching.html#caching

I have followed the documentation, added options, changed settings, etc., but have not been able to disable the cache. 

Sorry. 😢
